cli/tests/test_mcp.py

```
@@ -0,0 +1,275 @@
+import pytest
+import os
+import sys
+from unittest.mock import Mock, patch, MagicMock
```

Ruff (0.11.9)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)